packages/quantic/force-app/main/default/lwc/quanticResultList/e2e/quanticResultList.e2e.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @lwc/lwc/no-document-query */
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+import {testInsight, testSearch, expect} from './fixture';
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+const testFieldsToInclude = 'foo,bar,baz';
+const SEARCH_ORIGINCONTEXT = 'Search';
+const INSIGHT_ORIGINCONTEXT = 'InsightPanel';
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic result list ${useCase.label}`, () => {
+    test.describe('fieldsToInclude in the Search API request', () => {
+      test.use({
+        options: {
+          fieldsToInclude: testFieldsToInclude,
+        },
+      });
+      test('when performing a search should send the added fieldsToInclude', async ({
```

Comment:
nice 👍

Reply:
This one had me stumped for a while, all the fieldsToInclude are ADDED to the default fields sent by headless...

I was trying to remove fields from the default, and always seeing the same fields being sent.